### PR TITLE
Adding guide for democracy

### LINKED ISSUE #132: Guide: Voting for Referenda
We need a step-by-step guide for how one would vote on a referendum using the democracy module.

### PR DESCRIPTION
**Pull request by lsaether**
closes #132

### CODE REVIEW

**Comment by laboon** [comment]
@lsaether Is this still a draft?  Is it something that can be merged or merged after a bit of work?

**Comment by lsaether** [comment]
Not ready for review, just checking for conflicts.

**Comment by lsaether** [comment]
Ready for review.

File: docs/maintain-guides-democracy.md [hunk 1]
```diff
@@ -0,0 +1,236 @@
+---
+id: maintain-guides-democracy
+title: Participate in Democracy
+sidebar_label: Participate in Democracy
+---
+
+The public referenda chamber is one of the three bodies of on-chain governance
+as its instantiated in Polkadot and Kusama. The other two bodies are the
```
Comment (Swader) [inline docs/maintain-guides-democracy.md]:
```suggestion
as it's instantiated in Polkadot and Kusama. The other two bodies are the
```

File: docs/maintain-guides-democracy.md [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+
+Public referenda can be proposed and voted on by any token holder in the system
+as long as they provide a bond. After a proposal is made, others can agree with
+it by _seconding_ it and putting up tokens equal to the original bond. Every
+launch period, the most seconded proposal will be moved to the public referenda
+table will it can be voted upon. Voters who are willing to lock up their tokens
+for a greater duration of time can do so and get their vote amplified. For more
+details on the governance system please see [here](learn-governance).
+
+This guide will instruct token holders how to proposed and vote on public
```
Comment (Swader) [inline docs/maintain-guides-democracy.md]:
```suggestion
This guide will instruct token holders how to propose and vote on public
```

File: docs/maintain-guides-democracy.md [hunk 3]
```diff
@@ -0,0 +1,236 @@
...
+On Polkadot Apps you can use the "Democracy" tab to make a new proposal. In order
+to submit a proposal, you will need to submit what's called the preimage hash.
+The preimage hash is simply the hash of the proposal to be enacted. The easiest
+way to get the preimage hash is by clicking on the "Submit preimage" button and
+configuring the action that you are proposing.
+
+For example, if you wanted to propose that the account "Dave" would have a balance
+of 10 tokens your proposal may look something like the below image. The preimage
+hash would be `0xa50af1fadfca818feea213762d14cd198404d5496bca691294ec724be9d2a4c0`.
+You can copy this preimage hash and save it for the next step.
```
Comment (Swader) [inline docs/maintain-guides-democracy.md]:
```suggestion
You can copy this preimage hash and save it for the next step. There is no need to click Submit Preimage at this point, though you could. We'll go over that in the next section.
```

File: docs/maintain-guides-democracy.md [hunk 4]
```diff
@@ -0,0 +1,236 @@
...
+the transaction. You should now see your proposal appear in the "proposals"
+column on the page.
+
+![submit proposal](assets/democracy/submit_proposal.png)
+
+Now your proposal is visible by anyone who accesses the chain and others can
+second it or submit a preimage. However, it's hard to tell what exactly this
+proposal does since it shows the hash of the action. Other holders will not be
+able to make a judgement for whether they second it or not until someone submits
+the actual preimage for this proposal. The next step you will submit the preimage.
```
Comment (Swader) [inline docs/maintain-guides-democracy.md]:
```suggestion
the actual preimage for this proposal. In the next step you will submit the preimage.
```

File: docs/maintain-guides-democracy.md [hunk 5]
```diff
@@ -0,0 +1,236 @@
...
+update with the information for your already submitted proposal.
+
+![proposals updated](assets/democracy/proposals_updated.png)
+
+## Seconding a Proposal
+
+Seconding a proposal means that you are agreeing with the proposal and backing
+it with an equal amount of deposit as was originally locked. By seconding a 
+proposal you will move it higher up the rank of proposals. The most seconded
+proposal will be tabled as a referendum to be voted on every launch period.
```
Comment (Swader) [inline docs/maintain-guides-democracy.md]:
```suggestion
proposal - in value, not number of supporters - will be tabled as a referendum to be voted on every launch period.
```

File: docs/maintain-guides-democracy.md [hunk 6]
```diff
@@ -0,0 +1,236 @@
...
+Once successful you will see your second appear in the dropdown in the proposal
+details.
+
+![second result](assets/democracy/second_result.png)
+
+## Voting on a Proposal
+
+At the end of each launch period, the most seconded proposal will move to
+referendum. During this time you can cast a vote for or against the proposal.
+You may also lock up your tokens for a greater length of time to weigh your vote
```
Comment (Swader) [inline docs/maintain-guides-democracy.md:130]: Clarify what it means to lock tokens in this context - what can the locked tokens still do? Can they be double-locked for, e.g. identity, or council votes?

File: docs/maintain-guides-democracy.md [hunk 7]
```diff
@@ -0,0 +1,236 @@
...
+more strongly.
+
+To vote on a referendum, navigate to the "Democracy" tab of Polkadot Apps. Any
```
Comment (Swader) [inline docs/maintain-guides-democracy.md]:
```suggestion
To vote on a referendum, navigate to the ["Democracy" tab of Polkadot Apps](https://polkadot.js.org/apps/#/democracy/). Any
```

File: docs/maintain-guides-democracy.md [hunk 8]
```diff
@@ -0,0 +1,236 @@
...
+"Vote" to cast a vote for the referendum.
+
+If you would like to cast your vote for the proposal select the "Aye, I approve"
+option. If you would like to cast your vote against the proposal in referendum
+you will select "Nay, I do not approve" option. 
+
+The second option is to select your conviction for this vote. The longer you are
+willing to lock your tokens, the stronger your vote will be weighted. Unwillingness
+to lock your tokens means that your vote only counts for 10% of the tokens that
+you hold, while the maximum lock up of 128 days means you can make your vote count
```
Comment (Swader) [inline docs/maintain-guides-democracy.md]: Maximum is 6x at 256 days.

File: docs/maintain-guides-democracy.md [hunk 9]
```diff
@@ -0,0 +1,236 @@
...
+
+In the first input select the account you want to delegate to and in the second
+input select the amount of your conviction. Remember, higher convictions means
+that your vote will be locked longer. So choose wisely!
+
+![delegate](assets/democracy/delegate.png)
+
+After you send the delegate transaction, you can verify it went through by navigating
+to the "Chain State" tab and selecting the "democracy" and "delegations" options.
+You will see an output similar to below, showing the addresses which you have
```
Comment (Swader) [inline docs/maintain-guides-democracy.md]:
```suggestion
You will see an output similar to below, showing the addresses to which you have
```

File: docs/maintain-guides-democracy.md [hunk 10]
```diff
@@ -0,0 +1,236 @@
...
+delegated your voting power.
+
+![delegate state](assets/democracy/delegate_state.png)
+
+## Undelegate a Vote
+
+After some time if you decide to undelegate your vote from the delegated account,
+you must simply send the `undelegate` transaction from the account you wish to
```
Comment (Swader) [inline docs/maintain-guides-democracy.md]:
```suggestion
you must send the `undelegate` transaction from the account you wish to
```

File: docs/maintain-guides-democracy.md [hunk 11]
```diff
@@ -0,0 +1,236 @@
...
+clear.
+
+The easiest way to do this is from the "Extrinsics" tab of Polkadot Apps. Select
+the "democracy" pallet and the "undelegate" transaction type. Ensure that you are
+sending the transaction from the account you want to clear of delegations. Click
+"Submit transaction" and confirm.
```
Comment (Swader) [inline docs/maintain-guides-democracy.md:208]: Explain what happens to locked tokens in this case.

File: docs/maintain-guides-democracy.md [hunk 12]
```diff
@@ -0,0 +1,236 @@
...
+another.
+
+You can delegate your vote to another account and even attach a "Conviction"
+to the delegation. Navigate to the "Extrinsics" tab on Polkadot Apps and select
+the options "democracy" and "delegate". This means you are accessing the democracy
+pallet and choosing the delegate transaction type to send. 
+
+In the first input select the account you want to delegate to and in the second
+input select the amount of your conviction. Remember, higher convictions means
+that your vote will be locked longer. So choose wisely!
```
Comment (Swader) [inline docs/maintain-guides-democracy.md:179]: Are you locking tokens in the delegation, or to a specific decision that someone else voted for? I.e. if Alice votes YES on Proposal 1, and you delegate 20 tokens at 6x conviction, does this mean Alice's vote on Proposal 1 gets +120 immediately, or does she get the 120 to vote with on another referendum down the line? Which one? How do you decide what she can use those on?

File: docs/maintain-guides-democracy.md [hunk 13]
```diff
@@ -0,0 +1,236 @@
...
+Proxies can be used to vote on behalf of a stash account. Unlike delegation,
+the proxy is meant to act as a longer-term account that makes all the voting
+decisions for funds held in a different account. Delegation is a logical action,
+taken when you trust another account's judgement, while proxying is more of a
+recommended security practice for keeping your funds safe and using an active
+account with low funds instead.
+
+### Setting a proxy
+
+Setting a proxy is simple and involves only submitting a single transaction,
```
Comment (Swader) [inline docs/maintain-guides-democracy.md]:
```suggestion
Setting a proxy involves submitting a single transaction, 
```

**Review by Swader — CHANGES_REQUESTED** [review]
Nitpicks and typos, some clarifications needed.

Should we add disclaimers to pages that use Polkadot JS UI that the UI is only one of many possible implementations and that things might change in the future? Seems a bit vendor-lock-iny to constantly recommend a hosted UI without at least encouraging people to run it locally.

File: docs/maintain-guides-democracy.md [hunk 14]
```diff
@@ -0,0 +1,242 @@
...
+![second result](assets/democracy/second_result.png)
+
+## Voting on a Proposal
+
+At the end of each launch period, the most seconded proposal will move to
+referendum. During this time you can cast a vote for or against the proposal.
+You may also lock up your tokens for a greater length of time to weigh your vote
+more strongly. During the time your tokens are locked, you are unable to transfer
+them, however they can still be used for further votes. Locks are layered on top
+of each other, so a 8 week lock does not become a 15 week lock if you vote again
```
Comment (Swader) [inline docs/maintain-guides-democracy.md]:
```suggestion
of each other, so an eight week lock does not become a 15 week lock if you vote again
```

File: docs/maintain-guides-democracy.md [hunk 15]
```diff
@@ -0,0 +1,242 @@
...
+a week later, rather another 8 week lock is placed to extend the lock just 1 extra
```
Comment (Swader) [inline docs/maintain-guides-democracy.md]:
```suggestion
a week later, rather another eight week lock is placed to extend the lock just one extra
```

File: docs/maintain-guides-democracy.md (hunk 11 above)
Comment (Swader) [reply docs/maintain-guides-democracy.md:208]: No explanation about this?

File: docs/maintain-guides-democracy.md (hunk 12 above)
Comment (Swader) [reply docs/maintain-guides-democracy.md:179]: I think this should still be clarified.

**Review by Swader — CHANGES_REQUESTED** [review]
I'd like some more clarifications added to some parts. Also some nitpicks about spelling out numbers below 10 as per "standards".

**Comment by lsaether** [comment]
@Swader Can you review again? I think the points you wanted clarified have been resolved.

**Review by Swader — APPROVED** [review]
Perfect, this was a huge missing piece, I had no idea about this!

**Pull request merged**